ITGuru.FourWheels/ITGuru.FourWheels.Service/Extensions/ITaskExtensions.cs

```diff
@@ -0,0 +1,51 @@
+﻿using ITGuru.FourWheels.Data.DataModels;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace ITGuru.FourWheels.Service
```

Comment:
Is it on purpose that this it commented ?

Reply:
Yes. YOu'll not have the need for it, however, I need the systems from `Data` to actually form the logic.